NOTICE

```diff
@@ -4,3 +4,8 @@ Copyright 2017 The Apache Software Foundation
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
+
+----------------------------------------------------------------------------------------------------
+
+Circe CRC library
```

Comment:
Yes, and that is the reason why we don't mention it in the `LICENSE` file. The question here is whether we should mention the copyright or not. There are 2 different opinions in the replies on the voting thread, since the Circe repo doesn't include a `NOTICE` file. https://lists.apache.org/thread.html/c79ae36fdc1f734337a4f706117be87f4c4c568f87605c59ff4c3a96@%3Cgeneral.incubator.apache.org%3E

My guess is that adding the copyright would not do harm here.